src/gui/Gui.cc

```diff
@@ -53,6 +53,9 @@ std::unique_ptr<ignition::gui::Application> createGui(
   ignmsg << "Ignition Gazebo GUI    v" << IGNITION_GAZEBO_VERSION_FULL
          << std::endl;
 
+  // Set auto scaling factor for HiDPI displays
+  qputenv("QT_AUTO_SCREEN_SCALE_FACTOR", "1");
```

Comment:
how about checking to see if the env variable exists and only set the the variable if it does not? This allows users to override it if needed

Reply:
Added in 04488ee
Tested with:
```
$ ign gazebo
```
in which case the window auto scales, and
```
$ QT_AUTO_SCREEN_SCALE_FACTOR=0 ign gazebo
```
in which case the window does not auto scale, and
```
$ QT_AUTO_SCREEN_SCALE_FACTOR=2 ign gazebo
```
in which case the window auto scales, and
```
$ QT_AUTO_SCREEN_SCALE_FACTOR=-1 ign gazebo
```
in which case the window does not auto scale.